ZUGFeRD-Test/XRechnungUBLTests.cs

```diff
@@ -466,5 +466,29 @@ public void TestBuyerOrderReferenceLineId()
 
 			Assert.AreEqual(desc.TradeLineItems[0].BuyerOrderReferencedDocument.LineID, "6171175.1");
 		}
+        
+        
+        [TestMethod]
+        public void TestPartyIdentification_for_seller()
```

Comment:
can you use Pascal Case instead of snake case (i.e. TestPartyIdentificationForSeller)?

Can you add a second test case to test the case without party identification?

Reply:
What exactly is the second test supposed to test?

Reply:
@Tr1ckMan : my understanding was that PartyIdentification is only allowed for Sellers. Right?
If so, can you write a Seller without PartyIdentification and can you write another party and make sure that there is no PartyIdentification?